<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortSettingsWidget</class>
 <widget class="QWidget" name="SerialPortSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia portu szeregowego</string>
  </property>
  <property name="windowIcon">
   <iconset resource="guiIconsResources.qrc">
    <normaloff>:/icons/Computer-Hardware-Usb-Connector-icon.png</normaloff>:/icons/Computer-Hardware-Usb-Connector-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="portNameComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prędkość</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baudRateBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bity danych</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="dataBitsBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Parzystość</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bit stopu</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="parityBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="stopBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="guiIconsResources.qrc"/>
 </resources>
 <connections/>
</ui>
